src/game/SpellEffects.cpp

```diff
@@ -1149,6 +1149,13 @@ void Spell::EffectDummy(SpellEffectIndex eff_idx)
 
                     return;
                 }
+                case 32027:                                 // Expedition Flare
+                {
+                    if (!m_CastItem)                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
```

Comment:
That sounds very very weird, because (!m_CastItem) means NOT CastItem, so that means it only works if spell is not cast from an item? That sounds weird. I believe you, but it sounds like something is broken somehow.